packages/contentful-migrations/ui/1000000000004-create-c-editorial-migration.js

```diff
@@ -26,8 +26,18 @@ module.exports = async function (migration) {
     .omitted(false);
 
   cEditorial
-    .createField('secondary_headline')
-    .name('Secondary headline')
+    .createField('overline')
```

Comment:
Better put the overline before the headline so it's less confusing for the editor :)

Reply:
Done. 

And I have also removed all headlines in `m-text`. They are currently not rendered because the template does not support them.